openslides_backend/permissions/permissions.py

```diff
@@ -1,38 +1,54 @@
 # Code generated. DO NOT EDIT.
 
+from typing import Dict, List
+
 # Holds the corresponding parent for each permission.
-permissions = {
-    "agenda_item.can_see": "agenda_item.can_see_internal",
-    "agenda_item.can_see_internal": "agenda_item.can_manage",
-    "agenda_item.can_manage": None,
-    "list_of_speakers.can_see": "list_of_speakers.can_be_speaker",
-    "list_of_speakers.can_manage": None,
-    "list_of_speakers.can_be_speaker": None,
-    "meeting.can_manage_settings": None,
-    "meeting.can_manage_logos_and_fonts": None,
-    "meeting.can_see_frontpage": None,
-    "meeting.can_see_autopilot": None,
-    "meeting.can_see_livestream": None,
-    "meeting.can_see_history": None,
-    "projector.can_see": "projector.can_manage",
-    "projector.can_manage": None,
-    "tag.can_manage": None,
-    "motion.can_see": "motion.can_support",
-    "motion.can_manage_metadata": "motion.can_manage",
-    "motion.can_manage_polls": "motion.can_manage",
-    "motion.can_see_internal": "motion.can_manage",
-    "motion.can_create": "motion.can_manage",
-    "motion.can_create_amendments": "motion.can_manage",
-    "motion.can_manage": None,
-    "motion.can_support": None,
-    "assignment.can_see": "assignment.can_nominate_self",
-    "assignment.can_nominate_other": "assignment.can_manage",
-    "assignment.can_manage": None,
-    "assignment.can_nominate_self": None,
-    "mediafile.can_see": "mediafile.can_manage",
-    "mediafile.can_manage": None,
-    "user.can_see": "user.can_see_extra_data",
-    "user.can_see_extra_data": "user.can_manage",
-    "user.can_manage": None,
-    "user.can_change_own_password": None,
+permissions: Dict[str, List[str]] = {
```

Comment:
What do you think of putting all parents in the dict for each permission? We don't have a really deep structure and then the check would be easy again.

Reply:
Yes, I thought about that. Would have also been possible, but this way also works and we don't have extra data in this file ;)